.github/workflows/cicd.yaml

```diff
@@ -91,3 +91,11 @@ jobs:
         COMMIT_SHA: ${GITHUB_SHA}
         DOCKER_TAG: ${{ steps.make_tag.outputs.docker_tag }}
         DOCKER_PUSH_LATEST: ${{ github.event_name == 'push' }}
+    - run: make jcdc-deploy
+      if: github.event_name == 'push' # TODO: Remove when ready for per branch deployment
+      env:
+        REF: ${{ github.sha }}
+        JCDC_URL: https://jcdc.jul.run/run
+        JCDC_API_KEY: ${{ secrets.GOAT_JCDC_API_KEY }}
+        DEPLOY_HOSTNAME: foxtrot.jul.run # TODO: Update to {PR_NUMBER.}foxtrot.jul.run for per branch deployment
```

Comment:
You can leave this out, because the overlay has this. I suspect when we get to per-branch deployments, we will still not specify the hostname from the workflow, but just provide a prefix if necessary i.e. the overlay will still be the source of truth for the base hostname.

Reply:
ok

Reply:
ok